node/src/Commands.ts

```diff
@@ -1295,6 +1302,13 @@ export function createPExpireAt(
     return createCommand(RequestType.PExpireAt, args);
 }
 
+/**
+ * @internal
+ */
+export function createPExpireTime(key: string): command_request.Command {
```

Comment:
move up to be next to `createExpireTime`...?

Reply:
i was just following the list displayed on the valkey site.